src/runtime_src/core/tools/common/reports/ReportAiePartitions.cpp

```diff
@@ -97,6 +133,12 @@ writeReport(const xrt_core::device* /*_pDevice*/,
     return;
   }
 
+  const boost::property_tree::ptree pt_info = _pt.get_child("aie_partitions", empty_ptree);
+  if (pt_info.empty())
+    _output << "Total Column Utilization: N/A\n";
+  else
+    _output << boost::str(boost::format("Total Column Utilization: %s\n") % pt_info.get<std::string>("total_col_utilization"));
```

Comment:
Why don't you place N/A or an actual value in calculate_col_utilization()?
In L96 you can do something like:
`return (total_col_utilization==0) ? "N/A" : boost::str(boost::format("%.0f%%") % total_col_utilization);`

Reply:
I simplified this if suite to `_output << boost::str(boost::format("Total Column Utilization: %s\n") % _pt.get<std::string>("aie_partitions.total_col_utilization"));`, as total_col_utilization should always be there with "N/A" or a "%" string (I originally had that suite like that because I was imitating the empty ptree checks for partition data, my mistake!). Do we want to show "N/A" for "0%" utilization, though? This would only be if there are HW contexts on the device that are somehow all idle.